src/utils/function/clamp/clamp.test.ts

```
@@ -0,0 +1,35 @@
+
+  test("should return the maximum value if the value is greater than max", () => {
+    const result = clamp(15, 0, 10);
+    expect(result).toBe(10);
+  });
+
+  test("should throw a type error when value, min, or max is not a number", () => {
+    expect(() => clamp("5" as any, 1, 10)).toThrow(TypeError);
+  });
+
+  test("should throw a range error when value, min or max are passed greater than MAX_VALUE", () => {
+    expect(() => clamp(1e1000, 10, 1e20000)).toThrow(RangeError);
```

Biome

[error] This number literal will lose precision at runtime.

The value at runtime will be inf

(lint/correctness/noPrecisionLoss)